com.unity.render-pipelines.high-definition/Documentation~/Override-Physically-Based-Sky.md

```diff
@@ -27,6 +27,8 @@ Next, set the Volume to use **Physically Based Sky**. The [Visual Environment](O
 
 To change how much the atmosphere attenuates light, you can change the density of both air and aerosol molecules (participating media) in the atmosphere. You can also use aerosols to simulate real-world pollution or fog.
 
+**Note:** When Unity initializes a Physically Based Sky, it performs a resource-intensive operation which can cause the frame rate of your project to drop for a few frames. Once Unity has completed this operation, it stores the data in a cache to access the next time Unity initializes this volume. However, you may experience this frame rate drop if you have two Physically Based Sky volumes with different properties and switch between them.
```

Comment:
A question for the devs, is this the case just when going from one volume with a pb sky to a volume with another? Or does it happen whenever you enter a volume with a new pb sky?

Reply:
the second "Or does it happen whenever you enter a volume with a new pb sky?"
also you have it with a single volume simply when you start the game. i.e any time you init a new PBS sky you take a hit.

Reply:
The hitch will only occurs if some parameters are differents. Not all of the PBR sky parameters are used in the precompute so they don't all induce a frame drop.

Reply:
I think we can consider this as covered in the explanation by the "may experience"